features/datum.feature

```diff
@@ -124,13 +270,13 @@ Functionaliteit: leveren van een datum
       | burgerservicenummer                     | 555550002  |
```

Comment:
burgerservicenummer is nu niet meer nodig om op te nemen in de Gegeven

Reply:
verwijderd